ml4h/defines.py

```diff
@@ -161,3 +162,11 @@ def dataset_name_from_meaning(group: str, fields: List[str]) -> str:
     if group is None:
         return joined
     return group + HD5_GROUP_CHAR + joined
+
+
+class TensorGeneratorABC(ABC):
+    @abstractmethod
+    def __iter__(self) -> Iterator:
+        pass
```

Comment:
I think this might be equivalent to something a little cleaner
```
from collections.abc import Iterable
TensorGeneratorABC =  Iterator
```

Reply:
Done